Casks/s/stratoshark@dev.rb

```diff
@@ -0,0 +1,36 @@
+cask "stratoshark@dev" do
+  arch arm: "Arm", intel: "Intel"
+
+  version "0.9.0rc0,1340,2e3b802952cc"
+  sha256 arm:   "6b49beaae2f1e81cdbd2abcc5b6339a1132e200b8c310dd830010595293beacd",
+         intel: "3ec894824131be48835ac6757bb8866d1025210d140750e5f2bb0532c7701033"
+
+  url "https://www.wireshark.org/download/automated/osx/Stratoshark%20#{version.csv.first}-#{version.csv.second}-g#{version.csv.third}%20#{arch}%2064.dmg"
+  name "Stratoshark"
+  desc "System calls and log messages analyzer"
+  homepage "https://wiki.wireshark.org/Stratoshark"
+
+  livecheck do
+    url "https://www.wireshark.org/download/automated/osx/"
+    regex(/href=.*?Stratoshark%20v?(\d+(?:\.\d+)+(?:rc\d+)?)-(\d+)-g(\h+)%20#{arch}%2064.dmg"/i)
+    strategy :page_match do |page, regex|
+      page.scan(regex).map { |match| "#{match[0]},#{match[1]},#{match[2]}" }
+    end
+  end
+
+  auto_updates true
+  depends_on macos: ">= :big_sur"
+
+  app "Stratoshark.app"
+  binary "#{appdir}/Stratoshark.app/Contents/MacOS/extcap/falcodump"
+  binary "#{appdir}/Stratoshark.app/Contents/MacOS/extcap/sshdig"
+
+  uninstall quit:   "org.wireshark.Stratoshark",
+            delete: "/Applications/Stratoshark.app"
```

Comment:
```suggestion
  uninstall quit: "org.wireshark.Stratoshark"
```

The removal should already be handled by the `app` stanza.